src/connectors/index.ts

```diff
@@ -44,8 +45,9 @@ export const getNetworkLibraryByChain = (chainId: number) => {
     })
     return new Web3Provider(network.provider as any)
   } else if (chainId === ChainId.FUSE) {
+    const networkUrl = HOME_BRIDGE_CHAIN === ChainId.ROPSTEN ? ROPSTEN_NETWORK_URL : MAINNET_NETWORK_URL
```

Comment:
`HOME` is always the fuse :)
foreign it ropsten or main

Reply:
@leonprou I added a conditional for Lior he wanted to test the functionality with the home bridge set to ropsten, should i change that?

Reply:
there's home and foreign chains. Home is always fuse, while foreign is Ethereum or Ropsten (for tests).
so checking `HOME_BRIDGE_CHAIN === ChainId.ROPSTEN` doesn't make any sense. 
I see that came from env file. the env is
```
REACT_APP_CHAIN_ID="122"
REACT_APP_HOME_BRIDGE_CHAIN=1
```
`REACT_APP_CHAIN_ID` is represents the home chain
and `REACT_APP_HOME_BRIDGE_CHAIN` is Ethereum, so it's a foreign network actually
so should rename `REACT_APP_HOME_BRIDGE_CHAIN` to `REACT_APP_FOREIGN_BRIDGE_CHAIN` (and all the derived variables)

Reply:
@leonprou  I have renamed it, that name was causing the confusion, i will work better on naming

Reply:
cool, can you push the latest PR fixes?

Reply:
I have added the latest PR fixes